Manuteção de Campi

pedro-gandra opened the pull request:
Funcionalidade: Manutenção de Campi (inserir, alterar e deletar)

Tarefa trello: https://trello.com/c/sVbprAEQ/4-back-end-manuten%C3%A7%C3%B5es

Descrição: Desenvolvimento dos servlets usados na manutenção de campi, além do CampiModel e CampiRepository

pedro-gandra committed:
Colocando JDK compativel

alterando de JDK 12 para 8

pedro-gandra committed:
Merge remote-tracking branch 'origin/master' into feature/diario/campi

pedro-gandra committed:
Criação CampiModel

pedro-gandra committed:
Criação CampiModel

pedro-gandra committed:
Correção de erro

pedro-gandra committed:
CampiRepository e servlet deletar campi

pedro-gandra committed:
Alteração nos servlets

pedro-gandra committed:
Merge remote-tracking branch 'origin/dev-s1' into feature/diario/campi

pedro-gandra committed:
Mudando tudo pra nova estrutura

pedro-gandra committed:
Manutenção de Campi completa

pedro-gandra committed:
Melhoria no servlet AlterarCampi

ppassinho commented on app/src/java/diario/campi/model/Model.java:
```diff
@@ -0,0 +1,6 @@
+
+public abstract class Model {
+	public final static int ID_INDEFINIDO = -1;
+}
```
```suggestion
}

```

pedro-gandra committed:
Update app/src/java/diario/campi/model/Model.java

Co-Authored-By: Pedro César <pedrocesar1410@gmail.com>

LiviaLelis commented on app/src/java/diario/campi/repository/CampiRepository.java:
```diff
@@ -0,0 +1,93 @@
+    public boolean alterarCampi(String id, String nome, String cidade, String uf) throws NumberFormatException, SQLException, TransformerException, ParserConfigurationException {
+            
+                int adcs = 0;
+                String query = "UPDATE campi SET";
```
Query vulnerável

LiviaLelis commented on app/src/java/diario/campi/servlets/AlterarCampi.java:
```diff
@@ -0,0 +1,57 @@
+     protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        Connection conexao = ConnectionFactory.getDiario();
+        CampiRepository Rep = new CampiRepository(conexao);
```
variável começando com maiúscula

LiviaLelis commented on app/src/java/diario/campi/repository/CampiRepository.java:
```diff
@@ -0,0 +1,93 @@
+package diario.campi.repository;
```
Deve ser formatada com TABS

LiviaLelis commented on app/src/java/diario/campi/servlets/AlterarCampi.java:
```diff
@@ -0,0 +1,57 @@
+package diario.campi.servlets;
```
Deve ser formatado com TABS

LiviaLelis commented on app/src/java/diario/campi/servlets/DeletarCampi.java:
```diff
@@ -0,0 +1,48 @@
+package diario.campi.servlets;
```
TABS...

LiviaLelis commented on app/src/java/diario/campi/servlets/InserirCampi.java:
```diff
@@ -0,0 +1,49 @@
+package diario.campi.servlets;
```
TABS...

LiviaLelis commented on app/src/java/diario/campi/servlets/DeletarCampi.java:
```diff
@@ -0,0 +1,48 @@
+        String id = request.getParameter("id");
+            try{
+                if(Rep.deletarCampi(id))
+                    System.out.println("Deletado!");
```
Log desnecessário

LiviaLelis commented on app/src/java/diario/campi/servlets/DeletarCampi.java:
```diff
@@ -0,0 +1,48 @@
+@WebServlet(name = "DeletarCampi", urlPatterns = {"/diario/campi/deletar"})
+public class DeletarCampi extends HttpServlet {
+
+     protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
Sem reposta...

LiviaLelis commented on app/src/java/diario/campi/servlets/InserirCampi.java:
```diff
@@ -0,0 +1,49 @@
+@WebServlet(name = "InserirCampi", urlPatterns = {"/diario/campi/inserir"})
+public class InserirCampi extends HttpServlet {
+
+     protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
Sem resposta...

LiviaLelis commented on app/src/java/diario/campi/servlets/InserirCampi.java:
```diff
@@ -0,0 +1,49 @@
+        String uf = request.getParameter("uf");
+            try{
+                if(Rep.inserirCampi(nome, cidade, uf))
+                    System.out.println("Inserido!");
```
logs desnecessários

pedro-gandra committed:
Merge remote-tracking branch 'origin/dev-s1' into feature/diario/campi

pedro-gandra committed:
Correção de erros

Remoção de logs inuteis, correção na formatação, remoção de imports e arquivos não utilizados,  melhoria da query de alterar campi

pedro-gandra committed:
Merge remote-tracking branch 'origin/feature/diario/campi' into feature/diario/campi
